cypress/integration/plugins/observability-dashboards/3_trace_analytics_traces.spec.js

```diff
@@ -35,7 +35,7 @@ describe('Testing traces table', () => {
 
   it('Renders the traces table', () => {
     cy.contains(' (108)').should('exist');
-    cy.contains('03/25/2021 10:23:45').should('exist');
+    cy.contains('03/25/2021 17:23:45').should('exist');
```

Comment:
Will this change work only for UTC? What happens when we test dashboards on a cluster in other timezones?